src/components/reset-password/reset-password-controller.ts

```diff
@@ -48,7 +48,8 @@ export function resetPasswordPost(
       req.ip,
       sessionId,
       clientSessionId,
-      persistentSessionId
+      persistentSessionId,
+      req.session.user.withinForcedPasswordResetJourney
     );
```

Comment:
Minor suggestion, I think for readability you can extract this into its own var 
```
const {user, withinForcedPasswordResetJourney} = req.session.user
```
replacing req.session.user.withinForcedPasswordResetJourney >> with withinForcedPasswordResetJourney